tests/src/JIT/Methodical/tailcall_v4/smallFrame.il

```diff
@@ -358,7 +358,8 @@
     // Code size       7 (0x7)
     .maxstack  8
     //.line 114,114 : 3,16 ''
-    IL_0000:  tail. call       valuetype RetBuff RetBufferBug::TailCallee()
+//            tail.
```

Comment:
I might just delete these lines instead of commenting them out (unless we feel they add some value, in which case a short explanation re: why the `tail.` prefix has been removed would be good).